cypress/integration/learn/donate/donate-page-default.js

```diff
@@ -24,40 +14,33 @@ describe('Donate page', () => {
     cy.contains('Confirm your donation of $5 / month:').should('be.visible');
   });
 
-  it('Should have support section', () => {
+  it('Should have FAQ section', () => {
+    cy.contains('Frequently asked questions').should('be.visible');
+    cy.contains('How can I get help with my donations?').should('be.visible');
+    cy.contains('How transparent is freeCodeCamp.org?').should('be.visible');
+    cy.contains('How efficient is freeCodeCamp?').should('be.visible');
+    cy.contains('How can I make a one-time donation?').should('be.visible');
```

Comment:
There's no need for `should('be.visible')` after `contains`, since `contains` only returns something once it's in the DOM.